collector/src/execute.rs

```diff
@@ -130,6 +151,9 @@ struct BenchmarkConfig {
     /// directory that `Cargo.toml` is in.
     #[serde(default)]
     touch_file: Option<String>,
+
+    #[serde(default)]
```

Comment:
I think we should require a category for all benchmarks, rather than defaulting to secondary.

Reply:
Ok, sounds reasonable.